<ui version="4.0" >
 <class>IdForm</class>
 <widget class="QWidget" name="IdForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>IdForm</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Study:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="lineEdit" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Case:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="lineEdit_2" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>XML file:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="lineEdit_3" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
